### ISSUE TITLE
Enable/disable in stratum section stratum of config?

### ISSUE THREAD

**Issue by stone212** [paritytech/parity-ethereum#9785]
Looking at this page: https://wiki.parity.io/Configuring-Parity-Ethereum.html

I see the --stratum option on the command line but I don't see a toml option to enable stratum.  Is this one of those things that is right in front of me that I can't see?

**Comment by niklasad1** [comment]
@stone212 

It is sufficient to just add ```[stratum]``` in your `toml` file but we should add a enable/disable option to that for clarity!

Such as 
```toml
[stratum]
disable = true
```
```

**Comment by stone212** [comment]
@joshua-mir @niklasad1 

Just for clarity.  If you are already making a change I recommend making the one I posted.  It uses an already existing [category] and makes more logical sense (to me at least.)

**Comment by joshua-mir** [comment]
@stone212 Stratum is its own category because it has configuration options of its own. Ie, 

```
[stratum]
# Stratum Server will be running on port 8007.
port = 8007
# Secret for authorizing Stratum server for peers.
secret = "Yellow"
```

I agree that it might be better to have a enable/disable option though. Should I reopen as a feature request @niklasad1?

**Comment by niklasad1** [comment]
@joshua-mir Yes, please do

**Comment by joshua-mir** [comment]
Personally favor `disable = true/false` so existing configs with this quirk aren't broken. But the behavior is weird because it's disabled by default unless the [stratum] section is present 🤔

**Comment by Genysys** [comment]
should a different toml file be added for stratum or would it be wiser to add them to the exisiting toml files?

**Comment by joshua-mir** [comment]
Doesn't make sense not to just use the [stratum] section of the existing config - separate files should only be used for things like passwords

**Comment by niklasad1** [comment]
Closed by #10521
